sim_pipeline/gg_lens_pop.py

```diff
@@ -72,13 +74,18 @@ def draw_population(self):
         # Estimate the number of lensing systems
         num_lenses = self._lens_galaxies.deflector_number()
         num_sources = self._source_galaxies.galaxies_number()
-        num_lens_systems = int(num_lenses * num_sources)
-        print(num_lens_systems)
-        for _ in range(100):
-            source = self._source_galaxies.draw_galaxy()
+        num_sources_tested_mean = (testarea * self.f_sky * num_sources)/12960000
```

Comment:
I don't understand this line. Can you comment what happens here?

Reply:
to show the mean of source galaxies needed to tested within the test area.  (later draw a normal distribution around the mean for the number of source galaxies tested);
num_sources_tested_mean/ testarea = num_sources/ f_sky;
12960000= (3600)^2, since testarea is in arcsec^2 and f_sky is in deg^2.

Reply:
f_sky have error: num_sources_tested_mean = (testarea * num_sources)/ (12960000 * self.f_sky)